charts/firefly/values.yaml

```diff
@@ -4,8 +4,7 @@
 
 image:
   repository: ipac/suit
-  tag: lsst-stable
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
I don't like this, because DockerHub tags are not immutable.  It's very possible for someone to update an image and not update the tag.  If the current image *is* present, the pull will be near-instantaneous anyway.

Yes, I know DockerHub tags *should* be immutable, and that people *should not* rebuild the same tag with new contents.  We live in a fallen world.

Reply:
I'm pretty sure GPDF isn't going to reuse tags (he's bumping versions every time), and if he does, we can tell him not to?

Reply:
So, the concrete reason to not use `Always` is that it means that the deployment can't be restarted at the summit if the network is down. This seems bad, and I don't really want to have to manually change the pull policy of every image to `IfNotPresent` on the summit.